docs/use-cases/create-module.md

```diff
@@ -43,54 +42,187 @@ You can also watch this guide to creating a vision service module:
 
+In addition to determining which existing API namespace triplet to use when creating your module, you need to decide on a separate triplet unique to your model.
+
 A resource model is identified by a unique name, often called the {{< glossary_tooltip term_id="model-namespace-triplet" text="model namespace triplet" >}}, using the format: `namespace:repo-name:model-name`, where:
 
 - `namespace` is the [namespace of your organization](/cloud/organizations/#create-a-namespace-for-your-organization).
+  - For example, if your organization uses the `acme` namespace, your models must all begin with `acme`, like `acme:repo-name:mybase`.
+    If you do not intend to [upload your module](#upload-your-module-to-the-modular-resource-registry) to the [Viam registry](https://app.viam.com/registry), you do not need to use your organization's namespace as your model's namespace.
 - `repo-name` is the code repository (GitHub repo) that houses your module code.
+  - Ideally, your `repo-name` should describe the common functionality provided across the model or models of that module.
 - `model-name` is the name of the new resource model that your module will provide.
 
 For example, if your organization namespace is `acme`, and you have written a new base implementation named `mybase` which you have pushed to a repository named `my-custom-base-repo`, you would use the namespace `acme:my-custom-base-repo:mybase` for your model.
 
+More requirements:
+
+- Your model triplet must be all-lowercase.
+- Your model triplet may only use alphanumeric (`a-z` and `0-9`), hyphen (`-`), and underscore (`_`) characters.
+
 Determine the model name you want to use based on these requirements, then proceed to the next section.
 
-If you do not intend to [upload your module](/registry/upload/) to the [Viam registry](https://app.viam.com/registry), you do not need to use your organization's namespace as your model's namespace.
+#### API namespace triplet and model namespace triplet examples
+
+- The `rand:yahboom:arm` model and the `rand:yahboom:gripper` model use the repository name [yahboom](https://github.com/viam-labs/yahboom).
+  The models implement the `rdk:component:arm` and the `rdk:component:gripper` API to support the Yahboom DOFBOT arm and gripper, respectively:
+
+  ```json
+  {
+      "api": "rdk:component:arm",
+      "model": "rand:yahboom:arm"
+  },
+  {
+      "api": "rdk:component:gripper",
+      "model": "rand:yahboom:gripper"
+  }
+  ```
 
-See [Naming your model](/registry/#naming-your-model-namespacerepo-namename) for more information.
+- The `viam-labs:audioout:pygame` model uses the repository name [audioout](https://github.com/viam-labs/audioout).
+  It implements the custom API `viam-labs:service:audioout`:
+
+  ```json
+  {
+    "api": "viam-labs:service:audioout",
+    "model": "viam-labs:audioout:pygame"
+  }
+  ```
+
+The `viam` namespace is reserved for models provided by Viam.
 
 ### Write your new resource model definition
```

Comment:
Yes, it does feel overwhelming, but I just wanted to move this content over to here for now and then get into the details later. I edited the API triplet stuff quite a bit but not the model definition code stuff much at all.

At least one user indicates that examples are far and away the most useful thing, so whatever we do we want to make them as easy to find as possible. This could mean obvious prominent expanders, but yeah let's look at this in a separate PR?